meson.build

```diff
@@ -66,8 +66,13 @@ systemd = dependency('systemd', required: get_option('systemd'))
 
 if systemd.found()
   user_units_dir = systemd.get_pkgconfig_variable('systemduserunitdir')
-  install_data('./resources/waybar.service',
-    install_dir: user_units_dir)
+
+  configure_file(
+    configuration: config,
```

Comment:
I reverted this PR, CI says `config` is unknown :?a

Reply:
it seems I forgot to push that commit :man_facepalming: